### PR TITLE
Change methods name

### LINKED ISSUE #779: AVIF support: media library view
#### In `inc/classes/class-imagify-admin-ajax-post.php`
- rename `imagify_generate_webp_versions` to `imagify_generate_nextgen_versions`
- rename `imagify_generate_webp_versions_callback()` to `imagify_generate_nextgen_versions_callback()`
- rename `generate_webp_versions()` to `generate_nextgen_versions()` and update the method to use method from the optimization process to generate all next-gen versions needed

- rename `imagify_delete_webp_versions` to `imagify_delete_nextgen_versions`
- rename `imagify_delete_webp_versions_callback()` to `imagify_delete_nextgen_versions_callback()`
- rename `delete_webp_versions()` to `delete_nextgen_versions()` and update the method to delete all next-gen versions existing

#### In `inc/classes/class-imagify-files-list-table`
- rename `generate_webp_versions_button()` to `generate_nextgen_versions_button()`

#### In `inc/functions/admin.php`
- Update `get_imagify_admin_url()`, update the cases for `'generate-webp-versions` and `delete-webp-versions` to `generate-nextgen-versions` and `delete-nextgen-versions`

### Estimate the effort ✅
Effort [S]

### PR DESCRIPTION
**Pull request by Khadreal**
## Description

Changed methods name for media library view 

Fixes #779 

## Type of change

- Enhancement (non-breaking change which improves an existing functionality).

## Is the solution different from the one proposed during the grooming?
No

*Please describe in this section if there is any change to the groomed solution, and why.*

# Checklists

## Generic development checklist

- [x] My code follows the style guidelines of this project, with adapted comments and without new warnings.
- [ ] I have added unit and integration tests that prove my fix is effective or that my feature works.
- [x] The CI passes locally with my changes (including unit tests, integration tests, linter).
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] If applicable, I have made corresponding changes to the documentation. *Provide a link to the documentation.*

## Test summary

- [x] I triggered all changed lines of code at least once without new errors/warnings/notices.
- [ ] I validated all Acceptance Criteria of the related issues. (If applicable, provide proof).
- [ ] I validated all test plan the QA Review asked me to.

### CODE REVIEW

File: inc/classes/class-imagify-admin-ajax-post.php [hunk 1]
```diff
@@ -202,20 +202,20 @@ protected function optimize_missing_sizes( $media_id, $context ) {
 	 * @param  string $context  The context.
 	 * @return bool|WP_Error    True if successfully launched. A \WP_Error instance on failure.
 	 */
-	protected function generate_webp_versions( $media_id, $context ) {
+	protected function generate_nextgen_versions( $media_id, $context ) {
```
Comment (Miraeld) [inline inc/classes/class-imagify-admin-ajax-post.php:205]: :warning: this function has been refactored here with a different name : 
https://github.com/wp-media/imagify-plugin/blob/ffcbe66cfa06498264a8897d398357a7a9b97d7d/inc/classes/class-imagify-admin-ajax-post.php#L205

File: inc/classes/class-imagify-admin-ajax-post.php [hunk 2]
```diff
@@ -27,8 +27,8 @@ class Imagify_Admin_Ajax_Post extends Imagify_Admin_Ajax_Post_Deprecated {
 		'imagify_manual_optimize',
 		'imagify_manual_reoptimize',
 		'imagify_optimize_missing_sizes',
-		'imagify_generate_webp_versions',
-		'imagify_delete_webp_versions',
+		'imagify_generate_nextgen_versions',
```
Comment (Miraeld) [inline inc/classes/class-imagify-admin-ajax-post.php:30]: :warning: this function has been refactored here with a different name : 
https://github.com/wp-media/imagify-plugin/blob/ffcbe66cfa06498264a8897d398357a7a9b97d7d/inc/classes/class-imagify-admin-ajax-post.php#L30

File: inc/classes/class-imagify-files-list-table.php [hunk 3]
```diff
@@ -806,13 +806,13 @@ protected function reoptimize_buttons( $item ) {
...
 	/**
-	 * Prints a button to generate WebP versions if they are missing.
+	 * Prints a button to generate Next gen versions if they are missing.
 	 *
 	 * @since 1.7
 	 *
 	 * @param object $item The current item. It must contain at least a $process property.
 	 */
-	protected function generate_webp_versions_button( $item ) {
+	protected function generate_nextgen_versions_button( $item ) {
```
Comment (Miraeld) [inline inc/classes/class-imagify-files-list-table.php:815]: :warning: this function has been refactored here with a different name : 
https://github.com/wp-media/imagify-plugin/blob/ffcbe66cfa06498264a8897d398357a7a9b97d7d/inc/classes/class-imagify-files-list-table.php#L815

File: inc/classes/class-imagify-admin-ajax-post.php [hunk 4]
```diff
@@ -363,21 +363,21 @@ public function imagify_optimize_missing_sizes_callback() {
 	 *
 	 * @since 1.9
 	 */
-	public function imagify_generate_webp_versions_callback() {
+	public function imagify_generate_nextgen_versions_callback() {
```
Comment (Miraeld) [inline inc/classes/class-imagify-admin-ajax-post.php:366]: :warning: this function has been refactored here with a different name : 
https://github.com/wp-media/imagify-plugin/blob/ffcbe66cfa06498264a8897d398357a7a9b97d7d/inc/classes/class-imagify-admin-ajax-post.php#L366

**Comment by Khadreal** [comment]
@Miraeld I think I should assigned this https://github.com/wp-media/imagify-plugin/issues/779 to you, it seems you've already worked on it and https://github.com/wp-media/imagify-plugin/issues/778 by looking at your commits history

**Comment by Miraeld** [comment]
There is no need to re-assign it I think. 
From what I've seen on other PRs, we've been using `next_gen` notation instead of `nextgen`. 
As I needed few of these functions in an other PR, I had to rename them. I made my comment here to avoid either of us having conflicts.

**Comment by remyperona** [comment]
This PR needs conflict fixed before moving forward

**Comment by Khadreal** [comment]
While resolving the conflicts I noticed that the issue #779 was created based on develop branch and deature/avif branch has few things that aren't captured in the issue. 
Is it safe to rename all `next_gen` to `nextgen` ? @Tabrisrp

**Review by remyperona — APPROVED** [review]

**Pull request merged**